src/add-to-app/android/project-setup.md

```diff
@@ -322,6 +322,56 @@ dependencyResolutionManagement {
   }
 }
 
+```
+</br>
+
+### Kotlin DSL based Android Project
+
+After `aar` build, follow these steps to add flutter_module in Koltin DSL based Android Projects.
+
+Inlcude Flutter Module as dependecy in Android Project `app/build.gradle`
+<?code-excerpt title="MyApp/app/build.gradle.kts"?>
+```gradle
+android {
+    buildTypes {
+        release {
+          ...
+        }
+        debug {
+          ...
+        }
+        create("profile") {
+            initWith(getByName("debug"))
+        }
+}
+dependencies {
+  // ...
+  debugImplementation 'com.example.flutter_module:flutter_debug:1.0'
```

Comment:
this is a nit but let's use double-quotes in Groovy as well

Reply:
I added a suggestion for this.